Makefile

```diff
@@ -33,7 +38,12 @@ toitpkg: $(BUILD_DIR)/toitpkg
 TEST_FLAGS ?=
 .PHONY: test
 test: toitpkg $(GO_MOCKS)
-	tedi test -v -cover $(TEST_FLAGS) ./...
+	cmake -E env \
+	    TOITC_PATH=$(TOITC_PATH) \
+	    TOITLSP_PATH=$(TOITLSP_PATH) \
+			TOITVM_PATH=$(TOITVM_PATH)  \
```

Comment:
Why did you indent these options further?

Reply:
Accident: the top lines are indented with spaces, the remaining ones with tabs. In my editor tabs are only 2 spaces wide, so I didn't notice the difference.
done.